app_dart/lib/src/request_handlers/update_task_status.dart

```diff
@@ -5,14 +5,17 @@
 import 'dart:async';
 
 import 'package:gcloud/db.dart';
+import 'package:googleapis/firestore/v1.dart';
 import 'package:meta/meta.dart';
 
 import '../model/appengine/commit.dart';
 import '../model/appengine/task.dart';
+import '../model/firestore/task.dart' as f;
```

Comment:
Should we rename this import to something like 'fire' or 'firestore' for clarity?